test/fsharp-mode-font-tests.el

```diff
@@ -120,10 +120,6 @@
     (should (not (string-match fsharp-constructor-regexp test-string-2)))))
 
 ;; Operators
-(ert-deftest fsharp-operator-active-pattern-regexp-test ()
```

Comment:
No, I fear it doesn't. I realized active pattern parsing works considerably better with an [anchored matcher](https://github.com/fsharp/emacs-fsharp-mode/pull/117/files?diff=split#diff-ac75643809da8fc743af5fa432102e19R287), and I can't figure out how to pull it in to a constant and put it under test. Would love a suggestion if you've got any?